src/app/globals.css

```
@@ -262,6 +262,93 @@ body {
+/* 리사이즈 중 커서 전체 적용 */
+.resize-cursor {
+  cursor: col-resize;
+}
+
+.ProseMirror table {
+  table-layout: fixed;
+  width: 100%;
+}
+
+.ProseMirror table colgroup col {
+}
```

Stylelint (17.3.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)